integration-tests/config.go

```diff
@@ -6,23 +6,40 @@ import (
 	"time"
 )
 
-// Attributes that are unique to a validator. Allows us to map (part of) the set of uints to
-// a set of viable validators
+type chainID string
+type validatorID string
+
+const (
+	// Strings mapped to validator configs, literals must be two characters long for network config
```

Comment:
This is a requirement imposed by the fact that i was naming interfaces veth-$CHAIN_ID-$VAL_ID-out etc. We can change this to $CHAIN_ID-$VAL_ID-out, freeing up 5 characters and making it so that each string literal can be 5 chars or less.

Reply:
I'll also add a validation function to enforce this